service/src/event/ordering_task.rs

```diff
@@ -544,12 +544,8 @@ impl OrderingState {
             }
 
             event_cnt += 1;
-            let stream_cid = parsed_event
-                .id()
-                .expect("id must exist for non-init events");
-            let prev = parsed_event
-                .id()
-                .expect("prev must exist for non-init events");
+            let stream_cid = parsed_event.id();
+            let prev = parsed_event.id();
```

Comment:
This should be `prev()`.. it looks like we introduced a bug here that was released in v0.33 (made latest this week)... fortunately it's the startup task code and has probably never done anything. We don't have long streams currently so things shouldn't be getting picked up at startup, meaning we we're probably okay without doing any clean up, but I'm going to add a ticket to investigate how it got through/improve the tests.

Reply:
fixed in #516